core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -262,7 +262,7 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
   /**
    * Get a time parameter as seconds; throws a NoSuchElementException if it's not set. If no
    * suffix is provided then seconds are assumed.
-   * @throws NoSuchElementException
+   * @note Throws `NoSuchElementException`
```

Comment:
Oh, I think this may be resolved if `java.util.NoSuchElementException` is imported, or if the fully qualified name is used here. I favor the latter. That would be better; does it work?